spec/system/admin/admin_employee_manage_spec.rb

```diff
@@ -29,4 +30,15 @@
     page.driver.browser.switch_to.alert.accept
     expect(page).to have_content 'Employee account was deleted successfully'
   end
+
+  it 'checks if the rewads are listed in admin/employee panel' do
+    login_as(employee, scope: :employee)
+    random_reward = create(:reward)
+    visit 'employees/rewards'
+    click_button 'Buy'
```

Comment:
This test should test only a list of rewards, it will be better to just use here factorybot for order here

```suggestion
    create(:order)
```

the problem will be in the future when you change the page for buying rewards - for example, if you change in the future button `Buy` into `BUY`, this test will fail, but it shouldn't as this test if the rewards are listed in admin/employee panel

Reply:
I don't know if I can do it this way because it trigers the ' the numer_of_available_points > reward_price' constraint.